exchanges/okx/okx.go

```diff
@@ -1360,9 +1360,9 @@ func (e *Exchange) GetDepositWithdrawalStatus(ctx context.Context, ccy currency.
 }
 
 // SmallAssetsConvert Convert small assets in funding account to OKB. Only one convert is allowed within 24 hours
-func (e *Exchange) SmallAssetsConvert(ctx context.Context, currency []string) (*SmallAssetConvertResponse, error) {
+func (e *Exchange) SmallAssetsConvert(ctx context.Context, currencies []string) (*SmallAssetConvertResponse, error) {
```

Comment:
🦍 I'd take `currs` here
```suggestion
func (e *Exchange) SmallAssetsConvert(ctx context.Context, currs []string) (*SmallAssetConvertResponse, error) {
```